src/util/ssa_expr.cpp

```diff
@@ -82,7 +82,7 @@ bool ssa_exprt::can_build_identifier(const exprt &expr)
     return false;
```

Comment:
Commit message typo: "and probably shouldn't." -> "and probably shouldn't be part of it."